server/routes/addPatient/addPatientCompleted.test.ts

```diff
@@ -0,0 +1,130 @@
+    })
+
+    hospitalSearchService.getHospital.mockResolvedValue({
+      agencyId: 'SHEFF',
+      description: 'Sheffield Hospital',
+      longDescription: 'Sheffield Teaching Hospital',
+      agencyType: 'HOSP',
+      active: true,
+    } as Hospital)
+    prisonerSearchService.getPrisonerDetails.mockResolvedValue({
+      assignedLivingUnit: {
+        agencyId: 'MDI',
+        description: '1-2-015',
+      },
+      alerts: [
+        { alertType: 'T', alertCode: 'TCPA' },
+        { alertType: 'X', alertCode: 'XCU' },
+      ],
+      displayName: 'Smith, John',
+      formattedAlerts: [
+        {
+          alertCodes: ['XCU'],
+          classes: 'alert-status alert-status--controlled-unlock',
+          label: 'Controlled unlock',
+        },
+      ],
+      friendlyName: 'John Smith',
+      prisonerNumber: 'A1234AA',
+    } as PrisonerResultSummary)
+  })
+
+  afterEach(() => {
+    jest.resetAllMocks()
+  })
+  it('should load the prisoner add completed page', () => {
+    return request(app)
+      .get('/add-restricted-patient/prisoner-added?prisonerNumber=A1234AA&hospitalId=SHEFF')
+      .expect('Content-Type', /html/)
+      .expect(res => {
+        expect(res.text).toContain('John Smith has been added to Sheffield Hospital')
+        expect(res.text).toContain('John Smith’s prison number is A1234AA')
+      })
+  })
+
+  it('should render not found page if user missing privileges', () => {
+    mockJwtDecode.mockImplementation(() => ({ authorities: ['SEARCH_RESTRICTED_PATIENT'] }))
+    return request(app)
+      .get('/add-restricted-patient/prisoner-added?prisonerNumber=A1234AA&hospitalId=SHEFF')
+      .expect('Content-Type', /html/)
+      .expect(res => {
+        expect(res.text).toContain('Page not found')
+      })
+  })
+})
+describe('GET /prisoner-added - no session item (user jumped to page)', () => {
+  beforeEach(() => {
+    app = appWithAllRoutes({ production: false, services: { prisonerSearchService } })
+
+    hospitalSearchService.getHospital.mockResolvedValue({
+      agencyId: 'SHEFF',
+      description: 'Sheffield Hospital',
+      longDescription: 'Sheffield Teaching Hospital',
+      agencyType: 'HOSP',
+      active: true,
+    } as Hospital)
+    prisonerSearchService.getPrisonerDetails.mockResolvedValue({
+      assignedLivingUnit: {
+        agencyId: 'MDI',
+        description: '1-2-015',
+      },
+      alerts: [
+        { alertType: 'T', alertCode: 'TCPA' },
+        { alertType: 'X', alertCode: 'XCU' },
+      ],
+      displayName: 'Smith, John',
+      formattedAlerts: [
+        {
+          alertCodes: ['XCU'],
+          classes: 'alert-status alert-status--controlled-unlock',
+          label: 'Controlled unlock',
+        },
+      ],
+      friendlyName: 'John Smith',
+      prisonerNumber: 'A1234AA',
+    } as PrisonerResultSummary)
+  })
+
+  afterEach(() => {
+    jest.resetAllMocks()
+  })
+  it('should load the prisoner move completed page', () => {
```

Comment:
The description doesn't match the test...

Reply:
good spot, will fix in next PR